projects/vdk-core/tests/vdk/internal/builtin_plugins/ingestion/test_ingester_base.py

```diff
@@ -234,6 +238,44 @@ def test_pre_ingestion_operation():
         collection_id=shared_test_values.get("collection_id"),
         metadata=None,
     )
+    ingester_base._ingester.ingest_payload.assert_called_once()
+
+
+def test_pre_ingestion_updated_dynamic_params():
```

Comment:
I really like this test is it very clean.

Reply:
Thanks